package.json

```diff
@@ -32,7 +32,6 @@
     "eslint-plugin-jasmine": "^2.10.1",
     "eslint-plugin-prettier": "^3.0.0",
     "glob": "^7.1.3",
-    "handlebars": "^4.0.12",
```

Comment:
Let's leave handlebars for downstream folks that still use it :) Just the w3c profile won't import it anymore 🎉 I can ping downstream folks to see if they want to move to hyperHTML.